.github/workflows/release.yml

```
@@ -0,0 +1,88 @@
+# This workflow will build a Java project with Ant
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-ant
+
+name: openjverein official release
+
+on: workflow_dispatch
+
+jobs:
+  build:
+    name: Building release and upload to branch
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set up JDK 11 for x64
+      uses: actions/setup-java@main
```

Um den Wartungsaufwand zu verringern und mögliche unerwartete Seiteneffekte, bspw. durch Breaking Changes, auszuschließen, kann es hier langfristig bequemer sein für die stabilen Releases eine Version zu pinnen, bspw. die aktuelle `v3`.

```suggestion
      uses: actions/setup-java@v3
```